starlark/int.go

```diff
@@ -191,7 +191,7 @@ func (i Int) Hash() (uint32, error) {
 	return 12582917 * uint32(lo+3), nil
 }
 
-func (x Int) ThreeWayCompareSameType(v Value, depth int) (int, error) {
+func (x Int) Cmp(v Value, depth int) (int, error) {
 	y := v.(Int)
 	xSmall, xBig := x.get()
 	ySmall, yBig := y.get()
```

Comment:
Could you also add this comment to L201? Thanks.

return signum64(xSmall - ySmall), nil // safe: int32 operands

(It's not safe to use signum(x-y) in this way in general.)